src/js/modules/scrollTopArrow.js

```diff
@@ -0,0 +1,27 @@
+import { throttle } from "throttle-debounce";
+
+const SCROLL_DELAY = 250;
+const PAGE_TOP = 0;
+let button = document.getElementById("buttonScrollTop");
+let viewportHeight = document.documentElement.clientHeight;
```

Comment:
add viewportHeight tracking on window resize, so that you could change the height if user, for example, change mobile orientation to album and then back to vertical one

Reply:
Done